admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -194,14 +277,18 @@ export class AlertBannerModel extends HoistModel {
                 updatedBy: XH.getUsername()
             };
 
-        await XH.fetchJson({
-            url: 'alertBannerAdmin/setAlertSpec',
-            params: {value: JSON.stringify(value)}
-        }).track({
-            category: 'Audit',
-            message: 'Updated Alert Banner',
-            data: {active, message, intent, iconName, enableClose}
-        });
+        // {active, message, intent, iconName, enableClose}
+        // DEBUG
+        console.log('in save internal async');
+        this.saveBannerSpecAsync(value);
```

Comment:
this is *exactly* an example of where you do need to await per our discussion

Reply:
also, please be more careful not to leave the logging/commenting in the pushed code for PR